c7n/resources/elasticache.py

```diff
@@ -173,6 +177,12 @@ def process(self, clusters):
                 'Deleted ElastiCache replication group: %s',
                 replication_group)
 
+    def fetch_global_ds_rpgs(self):
+        global_rpgs = set()
+        rpgs = self.manager.get_resource_manager('elasticache-group').resources()
```

Comment:
augment=False we don't need tags here.

Reply:
Thanks, good catch!! Both the feedbacks should be addressed now.